contracts/Pool.sol

```diff
@@ -418,6 +454,25 @@ contract Pool is PoolFDT {
         _emitBalanceUpdatedEvent();
     }
 
+    /**
+        @dev View function to indicate if recipient is allowed to receive a transfer.
+             This is only possible if they have zero cooldown or they are passed their withdraw window.
+    */
+    function isReceiveAllowed(address to) public view returns (bool) {
+        (uint256 lpCooldownPeriod, uint256 lpWithdrawWindow) = globals.getLpParameters();
+        return block.timestamp > (withdrawCooldown[to] + lpCooldownPeriod + lpWithdrawWindow);
+    }
```

Comment:
One liner that was previously a call to `PoolLib`, now moved over from `PoolLib`. `lpCooldownPeriod` and `lpWithdrawWindow` fetched in one call.

Reply:
This is only used in `_checkTransfer`, so we can actually just move this logic into that function